lib/ramble/ramble/language/package_manager_language.py

```diff
@@ -41,6 +41,8 @@ def package_manager_variable(
     """
 
     def _define_package_manager_variable(pm):
+        import ramble.workload
```

Comment:
So previously it was working as this was implicitly imported?

Reply:
Yeah, I think it was just broken before because we never actually used this. But regardless, the import for this was not in the proper place.